src/components/CookiesBanner/index.jsx

```diff
@@ -0,0 +1,165 @@
+// @flow
+import Checkbox from '@material-ui/core/Checkbox'
+import Close from '@material-ui/icons/Close'
+import FormControlLabel from '@material-ui/core/FormControlLabel'
+import IconButton from '@material-ui/core/IconButton'
+import React, { useEffect, useState } from 'react'
+import { makeStyles } from '@material-ui/core/styles'
+import Link from '~/components/layout/Link'
+import { WELCOME_ADDRESS } from '~/routes/routes'
+import Button from '~/components/layout/Button'
+import { primary, mainFontFamily } from '~/theme/variables'
+import { loadFromStorage, saveToStorage } from '~/utils/storage'
+import type { CookiesProps } from '~/logic/cookies/model/cookie'
+import { COOKIES_KEY } from '~/logic/cookies/model/cookie'
+
+const useStyles = makeStyles({
+  container: {
+    backgroundColor: '#fff',
+    bottom: '0',
+    boxShadow: '0 2px 4px 0 rgba(212, 212, 211, 0.59)',
+    boxSizing: 'border-box',
+    display: 'flex',
+    justifyContent: 'center',
+    left: '0',
+    minHeight: '200px',
+    padding: '27px 15px',
+    position: 'fixed',
+    width: '100%',
+  },
+  content: {
+    maxWidth: '100%',
+    width: '830px',
+  },
+  text: {
+    color: primary,
+    fontFamily: mainFontFamily,
+    fontSize: '16px',
+    fontWeight: 'normal',
+    lineHeight: '1.38',
+    margin: '0 0 25px',
+    textAlign: 'center',
+  },
+  form: {
+    columnGap: '10px',
+    display: 'grid',
+    gridTemplateColumns: '1fr',
+    rowGap: '10px',
+    '@media (min-width: 960px)': {
+      gridTemplateColumns: '1fr 1fr 1fr',
+    },
+  },
+  formItem: {
+    alignItems: 'center',
+    display: 'flex',
+    justifyContent: 'center',
+  },
+  link: {
+    textDecoration: 'underline',
+    '&:hover': {
+      textDecoration: 'none',
+    },
+  },
+  close: {
+    position: 'absolute',
+    right: '12px',
+    top: '12px',
+  },
+})
+
+const CookiesBanner = () => {
+  const classes = useStyles()
+
+  const [showBanner, setShowBanner] = useState(false)
+  const [localNecessary, setLocalNecessary] = useState(true)
+  const [localAnalytics, setLocalAnalytics] = useState(false)
+
+  useEffect(() => {
+    async function fetchCookiesFromStorage() {
+      const cookiesState: CookiesProps = await loadFromStorage(COOKIES_KEY)
+      if (cookiesState) {
+        const { acceptedNecessary, acceptedAnalytics } = cookiesState
+        setLocalAnalytics(acceptedAnalytics)
+        setLocalNecessary(acceptedNecessary)
+        setShowBanner(acceptedNecessary === false)
+      } else {
+        setShowBanner(true)
+      }
+    }
+    fetchCookiesFromStorage()
+  }, [])
+
+  const acceptCookiesHandler = async () => {
+    const newState = {
+      acceptedNecessary: true,
+      acceptedAnalytics: true,
+    }
+    await saveToStorage(COOKIES_KEY, newState)
```

Comment:
as mentioned in the call we had, the consent should be stored as a cookie (you can use `js-cookie` package for this) because we have to renew it (IIRC its 365 days for positive and 7 days for negative, but better to double check with legal)
cc @lukasschor

Reply:
done